source/Planet.h

```diff
@@ -160,6 +164,7 @@ class Planet {
 	mutable Sale<Outfit> outfitter;
 
 	const Government *government = nullptr;
+	float landingSpeed = 0.f;
```

Comment:
```suggestion
	float landingSpeed = 0.f;
```
shouldnt it be defined to normal? 1.0f?

Reply:
This is flat landing speed that is added to the other one